runtime-modules/council/src/lib.rs

```diff
@@ -204,6 +203,11 @@ pub trait Trait: system::Trait {
         membership_id: &Self::MembershipId,
         account_id: &<Self as system::Trait>::AccountId,
     ) -> bool;
+
+    fn is_account_free_of_conflicting_stakes(
```

Comment:
This needs to be on `StakeHandler` for consistency please review that trait.

Reply:
Done